### ISSUE TITLE
Accessibility Bug: There is no text alternative in the form of tooltip provided for the ‘Close’ button in the message strips.

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3121]
Fundamental core component: :"Message Strip"

Description: There is no text alternative in the form of tooltip provided for the ‘Close’ button in the message strips.

Expected: The text alternative should be provided for every non-text content element which fulfills the purpose of the content.

![image](https://user-images.githubusercontent.com/32538291/91556276-1956dd00-e950-11ea-95fb-b796662a1dc0.png)

Note: - Issue persists for all the close buttons present within the ‘Message strip’ control.
